.github/workflows/package.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Setup Go
       uses: actions/setup-go@v2
       with:
-        go-version: 1.17
+        go-version: 1.19
```

Comment:
Seems like I forgot to update this workflow. Would you update it to `setup-go@v3` and usage of the `go.mod` file like the other workflow? I can also do this separately if you like.

Reply:
Good point, changed it. Nice, I wasn't aware of that feature!